php-checks/src/main/java/org/sonar/php/checks/PHPDeprecatedFunctionUsageCheck.java

```diff
@@ -98,8 +108,17 @@
       .add(PARSE_STR_FUNCTION)
       .add(ASSERT_FUNCTION)
       .add(DEFINE_FUNCTION)
+      .add(STREAM_FILTER_APPEND_FUNCTION)
```

Comment:
what about `stream_filter_prepend` as described in the ticket ?!
I think that for this case, I would prefer to raise an issue every time `string.strip_tags` is used literally, whatever the context.

Reply:
Good idea, let's raise issue on each string literal, no FN and change to have a FP is super low.